src/iuse.cpp

```diff
@@ -2600,7 +2635,7 @@ int iuse::extinguisher(player *p, item *it, bool, const tripoint& )
 
 int iuse::rm13armor_off(player *p, item *it, bool, const tripoint& )
 {
-    if (it->charges < it->type->charges_to_use()) {
+    if (it->ammo_remaining() < it->type->charges_to_use()) {
```

Comment:
Could use `ammo_required()` or even better you could yank the `ammo_sufficient()` from my recent NPC PR that your currently testing